packages/react-component-library/src/primitives/FloatingBox/FloatingBox.tsx

```diff
@@ -26,7 +26,7 @@ export interface FloatingBoxProps extends PositionType, ComponentWithClass {
 }
 
 export const FloatingBox = forwardRef(
-  (props: FloatingBoxProps, ref?: React.Ref<HTMLDivElement>) => {
+  (props: FloatingBoxProps, ref?: React.Ref<any>) => {
```

Comment:
Does this need to be `any`? Does `HTMLElement` work?

Reply:
I couldn't get this to play nice with anything other than any because there seems to be some intermediary type of LegacyRef being passed (I think it's coming from the the Tether library implementation).